### ISSUE TITLE
Media Library: Incorrect use of `aria-label`

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#7574]
## Bug Description

Images and videos in the media library incorrectly have `aria-label` attributes.

Both should only have `alt` attributes.

Offending code:

https://github.com/google/web-stories-wp/blob/4c891bd2b6b10815c72b06d7bbccd15718da79b1/assets/src/edit-story/components/library/panes/media/common/innerElement.js#L135-L167

## Expected Behaviour

No usage of `aria-label`. Just `alt`

## Screenshots

![Screenshot 2021-05-18 at 23 14 35](https://user-images.githubusercontent.com/841956/118724516-fbf3d700-b82e-11eb-84e0-dd036cc8845c.png)

![Screenshot 2021-05-18 at 23 14 15](https://user-images.githubusercontent.com/841956/118724524-fd250400-b82e-11eb-935a-4ed06cdc697c.png)

## Additional Context

- Plugin Version: 1.7
- WordPress Version: 5.7
- Operating System: macOS
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

* Remove the two `aria-label` props in the code section shared above

**Comment by spacedmonkey** [comment]
As far I know, video tags do not support alt tags. It should be removed on video tags.

**Comment by swissspidy** [comment]
I just found https://www.powermapper.com/tests/screen-readers/media/ and it seems like `video[title]` and `video[aria-label]` are actually the options with most support.

Given that we use `video[title]` on the frontend, we can use `title` for videos in the media library as well.

**Comment by Sreelayavuyyuru** [comment]
As far as I understood, you wanted to remove 'aria-label' and just let alt be. 

I am sorry if I am wrong anywhere. I am fairly new to open-source development, I am still exporing.

**Comment by Sreelayavuyyuru** [comment]
I created a draft PR to solve this issue, I am looking forward for any improvement suggestions.
